pkg/image/trigger/deploymentconfigs/deploymentconfigs_test.go

```diff
@@ -91,6 +91,28 @@ func TestDeploymentConfigReactor(t *testing.T) {
 		},
 
 		{
+			// no container, second run
+			tags: []fakeTagResponse{{Namespace: "other", Name: "stream-1:1", Ref: "image-lookup-2", RV: 3}},
+			obj: testDeploymentConfig([]deployapi.DeploymentTriggerImageChangeParams{
+				{
+					Automatic:          true,
+					From:               kapi.ObjectReference{Name: "stream-1:1", Namespace: "other", Kind: "ImageStreamTag"},
+					ContainerNames:     []string{"test"},
+					LastTriggeredImage: "image-lookup-1",
+				},
+			}, nil),
```

Comment:
@smarterclayton I have extended the tests to verify substituting images as well. ptal